validate.py

```diff
@@ -12,8 +13,28 @@
 
 
 def validate(pep_seq, mhc_name, mod_type=None, mod_pos=None):
+    # Thanks to Austin Crinklaw
+    pattern = re.compile(r"[^A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|X|Y]", re.IGNORECASE)
+    has_amino_acids = pattern.findall(pep_seq)
+    if has_amino_acids:
+        return f"Peptide sequence {pep_seq} has characters {has_amino_acids} that are not amino acids"
+    #if mod_pos and not mod_type:
+        #return "Modificiation position provided but no modification type"
+    #if mod_type and not mod_pos:
+        #return "Modification type provided but not modification position"
     if mod_pos:
-        statement = validate_mod_pos(pep_seq, mod_pos)
+        modifications = mod_pos.replace(" ", "")
+        modifications = modifications.split(",")

```

Comment:
This would be a little cleaner as `modifications = mod_pos.replace(" ", "").split(",")`